docs/pages/use-cases/WalletQueries/wallet07.mdx

```diff
@@ -2,7 +2,108 @@ import { Callout, Code, Tabs, Steps } from "nextra/components"
 
 ## 7. Query Active Stake Pools
 
-[Staking](https://cardano.org/stake-pool-delegation/) is a big part of the Cardano ecosystem. To get started, the first step is to identify the active stake pools. Our next SQL query retrieves information about active Cardano staking pools, retrievign the pool's ID, name and creation date. 
+[Staking](https://cardano.org/stake-pool-delegation/) is a big part of the Cardano ecosystem. The more healthy stake pools in operation, the more secure and resilient the network becomes. 
+
+### 7.1 Query all Stake Pools
+
+To get started, the first step is to identify all stake pools. This SQL query retrieves information about active Cardano staking pools, retrieving the pool's ID, name and creation date. 
```

Comment:
"This SQL query retrieves information about active Cardano staking pools".
All stake pools also include retired, so can "active" be removed? This is the next query :)